python/res/job_queue/workflow_runner.py

```diff
@@ -41,7 +41,14 @@ def __runWorkflow(self):
 
     def isRunning(self):
         """ @rtype: bool """
-        return self.__workflow.isRunning()
+        workflow_running = self.__workflow.isRunning()
```

Comment:
Would if be easier to read as:

```python
if self.__workflow.isRunning():
    return True
return self._workflow_job is not None and not self._workflow_job.done()
```

or something similar?

Reply:
Indeed! There are many ways to skin this beast.